src/Database/Driver.php

```diff
@@ -303,9 +296,7 @@ public function quote($value, $type = PDO::PARAM_STR): string
      */
     public function supportsQuoting(): bool
     {
-        $this->connect();
-
-        return $this->_connection->getAttribute(PDO::ATTR_DRIVER_NAME) !== 'odbc';
```

Comment:
Should this have a deprecation warning?

Reply:
Added for all support functions this replaces.